src/components/StudyList/StudyTable.js

```diff
@@ -46,6 +46,12 @@ const TableValue = ({row, col, title}) => {
             ' complete'}
         </Link>
       );
+    case 'release':
+      const verson = row[col].node ? row[col].node.version : 'Not Published';
+      const time = row[col].node
+        ? longDate(new Date(row[col].node.createdAt))
+        : null;
+      return <span title={'Published on ' + time}>{verson}</span>;
```

Comment:
An inverted `Popup` may work better here.
Can we also make it a link out to the coordinator?